language/stdlib/transaction_scripts/peer_to_peer_with_metadata.move

```diff
@@ -1,6 +1,13 @@
 use 0x0::LibraAccount;
 
-fun main<Token>(payee: address, auth_key_prefix: vector<u8>, amount: u64, metadata: vector<u8>) {
-  if (!LibraAccount::exists(payee)) LibraAccount::create_unhosted_account<Token>(payee, copy auth_key_prefix);
-  LibraAccount::pay_from_sender_with_metadata<Token>(payee, amount, metadata)
+fun main<Token>(
```

Comment:
@christinacelee / @thefallentree: let me know if this approach makes sense to you from a client perspective.. I figured that it would be easiest/most consistent to have one payment script that can do everything (i.e., both travel rule and non-travel rule payments). We can add separate scripts without parameters like `metadata_signature` and `auth_key_prefix` in the future if they are useful.

Reply:
Also: would you mind taking a look at the JSON-rpc changes to make sure I didn't do anything silly there?